### PR TITLE
Update tzkt to match flextesa

### PR DESCRIPTION
**Pull request by AlirezaHaghshenas**
# 🌮 Taqueria PR

## 🪁 Description

_Explain it to me like I'm 5! Keep this short and descriptive_

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [ ] ⛱️ I have completed this PR template in full and updated the title appropriately
- [ ] ⛵ My code builds cleanly, and I have manually tested the changes
- [ ] 🏄‍♂️ Another team member has built this branch and done manual testing on the change
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🦀 Automated tests have been written and added to this PR
- [ ] 🐬 I have commented my code, particularly in hard-to-understand areas
- [ ] 🤿 Corresponding changes have been made to all documentation
- [ ] 🐚 Required changes to the VScE have been made
- [ ] 🪸 Required updates to scaffolds have been made
- [ ] 🚢 The release checklist has been completed

## 🛩️ Summary of Changes

_Please include a summary of the changes to the codebase._ _Please also include relevant motivation and context for the change_
_(e.g. what was the existing behaviour, why did it break, etc.)_

## 🎢 Test Plan

_Please describe the testing strategy and plan for this PR. Keep this lightweight and anticipate any testing challenges_

## 🛸 Type of Change

- [ ] 🧽 Chore ➾
- [ ] 🛠️ Fix ➾
- [ ] ✨ Feature ➾
- [ ] 👷 Refactor ➾
- [ ] 🧪 Pre-Release ➾
- [ ] 🚀 Release ➾

### CODE REVIEW

File: taqueria-flextesa-manager/lib.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const startMininet = (parsedArgs: SanitizedArgs.t) => {
 		...getBootstrapFlags(parsedArgs),
 		'--until-level 200_000_000',
 		// TODO: Find a way of mapping protocol hash to protocol kind
-		`--protocol-kind "Alpha"`,
+		`--protocol-kind "Kathmandu"`,
```
Comment (jchenche) [inline taqueria-flextesa-manager/lib.ts:50]: May I suggest we put it at the top and use a const variable?

e.g. `const PROTOCOL = "Kathmandu"`

File: taqueria-flextesa-manager/lib.ts [hunk 2]
```diff
@@ -68,7 +68,7 @@ export const importAccounts = (opts: SanitizedArgs.t) => {
 			if (accountName === 'default') return retval;
 			const account = accountDetails as SandboxAccountConfig.t;
 			// const protocol = getSandboxProtocol(opts)
-			const protocol = 'ProtoALphaAL';
+			const protocol = 'PtKathmankSp';
```
Comment (jchenche) [inline taqueria-flextesa-manager/lib.ts:71]: May I suggest we put it at the top and use a const variable?

e.g. `const PROTOCOL_HASH = "PtKathmankSp"`

File: taqueria-flextesa-manager/lib.ts (hunk 1 above)
Comment (AlirezaHaghshenas) [reply taqueria-flextesa-manager/lib.ts]: Sure, done

File: taqueria-flextesa-manager/lib.ts (hunk 2 above)
Comment (AlirezaHaghshenas) [reply taqueria-flextesa-manager/lib.ts]: Done

File: taqueria-flextesa-manager/lib.ts [hunk 3]
```diff
@@ -8,6 +8,9 @@ import * as SanitizedArgs from './SanitizedArgs';
 
 import { Config, SandboxAccountConfig } from '@taqueria/protocol/taqueria-protocol-types';
 
+const protocolIdentifier = 'PtKathmankSp';
```
Comment (jchenche) [inline taqueria-flextesa-manager/lib.ts:11]: What do you think of MACRO style naming for these global constants? It'd distinguish them from normal local constants which using camel case like what you have here. This makes it easy to know which is which when reading code that uses them.

Comment (jchenche) [reply taqueria-flextesa-manager/lib.ts:11]: This is what I've adopted throughout my code at least.

Comment (jchenche) [reply taqueria-flextesa-manager/lib.ts:11]: e.g. `PROTOCOL_HASH`

Comment (AlirezaHaghshenas) [reply taqueria-flextesa-manager/lib.ts]: Yes. Sure, Done

**Review by jchenche — APPROVED** [review]

**Review by mweichert — APPROVED** [review]

**Review by hu3man — APPROVED** [review]

**Pull request merged**